_15thnight/core.py

```diff
@@ -130,3 +131,26 @@ def send_out_resolution(need):
                 body=body
             )
         )
+
+    # Check if alert is closed, if so, send out resolution notices
+    send_alert_resolution_notice.apply_async(args=[need])
```

Comment:
This doesn't need to be async. Delete lines 136-140 please. It can all be one function. `Alert.is_open` is not expensive and will work quickly. Things that need to be async are actions like communicating to a remote web or mail server. Communicating to a database (probably local) that the app has binded to perfirms fine.